application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -228,11 +225,17 @@ $doc.display($field)
     : #displayDate('endDate')#showvalidationmessage("MoccaCalendar.Code.MoccaCalendarEventModificationClass.validation.endDateBeforeStartDate")#showvalidationmessage("MoccaCalendar.Code.MoccaCalendarEventModificationClass.validation.endDateNotDate")
     ; &lt;label for="${editOnePrefix}_description"&gt;$escapetool.xml($doc.displayPrettyName('description', false, false))&lt;/label&gt;
     : $doc.display('description')
-
+    
+    #set($dateFormat = $!xwiki.getClass("MoccaCalendar.Code.MoccaCalendarEventModificationClass").get("startDate").getProperty('dateFormat').value)
+    #if("$!dateFormat" == "")
+      ## If the date format set from the current xwiki is not set, a default value will be provided.
+      #set($dateFormat = $xwiki.getXWikiPreference('dateformat', 'yyyy/MM/dd HH:mm'))
+    #end
     &lt;input type="hidden" name="classname" value="MoccaCalendar.Code.MoccaCalendarEventModificationClass" /&gt;
     &lt;input type="hidden" id="modificationIsNew" name="modificationIsNew" value="$modifiedEventObjectIsNew" /&gt;
     &lt;input type="hidden" id="eventOrigStartDate" name="${editOnePrefix}_eventOrigStartDate" value="$escapetool.html($instanceStartStr)" /&gt;
     &lt;input type="hidden" name="eventDoc"  value="$escapetool.html($services.model.serialize($doc.getDocumentReference(),'default'))" /&gt;
+    &lt;input type="hidden" id="used-format" name="used-format" value="$dateFormat" /&gt;
```

Comment:
Isn't this input send along with a form, you need it there? Maybe add it on the parent div (or in a span) in a `data-*` attribute